riff-raff/app/assets/javascripts/collapsing.coffee

```diff
@@ -13,6 +13,7 @@ setupCallbacks = ->
 
 
 $ ->
+  setupCallbacks()
```

Comment:
I don't quite understand why this is needed (I also don't think I want to... ☕ 😱 )

Reply:
The section below was rerunning it after refreshes, but I am now using it on a page that doesn't have auto-refresh and so the callbacks were never being instantiated. Rewriting all this in react would be fun hey?